### PR TITLE
Update Aerosols

### PR DESCRIPTION
**Pull request by kdorheim**
Related to #700, update the aerosol scalers

### CODE REVIEW

**Comment by kdorheim** [comment]
Implement aerosol forcing parameters to account for the new analysis by Zelinka et al. (2023)

> Hector 3.0 is calibrated to forcing values in the AR6, which were informed by analysis of ESM model results as documented in Smith et al. (2021). We adjust aerosol forcing parameters in this version of the model from the values documented in Smith et al. (2021) to account for the new analysis by Zelinka et al. (2023). Zelinka et al. find, due to a coding error, a higher absorbing forcing, which is largely countered by a larger cloud indirect forcing. We therefore scale the magnitude of BC, other cooling aerosols, and F_ACI by the ratio of average CMIP6 model values from Zelinka et al. (2023, Table 2) and Smith et al. (2020) (Table 6). These changes largely offset each other, but result in a slightly higher net negative forcing  for the current day and also result in slightly different total aerosol forcing pathways over time.
``` 
[Note two different references. Smith et al. 2021, and Smith et al. 2020. No relation :)
 
The resulting changes are to the following 5 aerosol constants:
 
SI Eq 12 -ρ𝐵𝐶 (new) = ρ𝐵𝐶(old) * ( (0.37+0.07) / (0.28 + 0.07) )**
 
SI Eq 13 through SI Eq 15 (for all the reflecting forcings) multiply by ( 0.65/0.63 )
 
SI Eq 16 -ρaci (new) = ρaci (old) * ( 0.88/ 0.81 )
 
The result will be a about a 5% larger net negative forcing, and a larger positive ARI from BC.
 
 
 
** Note that for BC we have assumed that the bulk of long-wave forcing is also due to BC absorption. Note that, while the long-wave forcing in Zelinka et al. (2023) and Smith et al. (2020) differ, this difference is not due to a coding change, but due to a different methodology in Zelinka et al. regarding how land surface warming is treated (Zelinka pers. communication). Because this is a definitional difference, we assume the LW forcing is the same between the two studies and allocate this to BC. 
 
 
 
Smith et al. (2021)
Smith, C. J., Kramer, R. J., Myhre, G., Alterskjær, K., Collins, W., Sima, A., Boucher, O., Dufresne, J.-L., Nabat, P., Michou, M., Yukimoto, S., Cole, J., Paynter, D., Shiogama, H., O'Connor, F. M., Robertson, E., Wiltshire, A., Andrews, T., Hannay, C., Miller, R., Nazarenko, L., Kirkevåg, A., Olivié, D., Fiedler, S., Lewinschal, A., Mackallah, C., Dix, M., Pincus, R., and Forster, P. M.: Effective radiative forcing and adjustments in CMIP6 models, Atmos. Chem. Phys., 20, 9591–9618, [https://doi.org/10.5194/acp-20-9591-2020](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdoi.org%2F10.5194%2Facp-20-9591-2020&data=05%7C02%7Ckalyn.dorheim%40pnnl.gov%7C3335fc7b9a4d4e48b7ce08dc0cab8f31%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C638399178177466654%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cXkqoDl%2FNlnN2H%2BAU%2F8prdBLwXYPthnR8%2FBAkBEYr14%3D&reserved=0), 2020.
 
Zelinka et al. (2023)
Zelinka, M. D., Smith, C. J., Qin, Y., and Taylor, K. E.: Comparison of methods to estimate aerosol effective radiative forcings in climate models, Atmos. Chem. Phys., 23, 8879–8898, [https://doi.org/10.5194/acp-23-8879-2023](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdoi.org%2F10.5194%2Facp-23-8879-2023&data=05%7C02%7Ckalyn.dorheim%40pnnl.gov%7C3335fc7b9a4d4e48b7ce08dc0cab8f31%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C638399178177623939%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eI1Ocsarm4C2P3nsvVDhvJ%2FUzKooY%2F2%2FB1GOyYuXdwU%3D&reserved=0), 2023.
```

**Comment by kdorheim** [comment]
Minor changes in global temp 
image

Changes are more noticeable for the specific aerosols 
image

File: inst/include/forcing_component.hpp [hunk 1]
```diff
@@ -120,7 +120,7 @@ class ForcingComponent : public IModelComponent {
 
   // Aerosol parameters for aerosol-cloud interactions (RFaci) see equation
   // Equation 7.SM.1.2 of IPCC AR6
-  double const aci_beta = 2.09841432; // (W m-2) IPCC AR6 7.SM.1.3.1
+  double const aci_beta = 2.279759; // see Dorheim et al. in prep
```
Comment (bpbond) [inline inst/include/forcing_component.hpp:123]: Add a TODO or a GitHub issue so we remember to come back and update this

File: inst/input/hector_ssp126.ini [hunk 2]
```diff
@@ -51,7 +51,7 @@ soil_c=917   		; initial soil pool, Pg C
 ;nonpf.soil_c=609
 
 ; Permafrost is active by default in Hector v3
-; See Woodard et al. 2021, 10.5194/gmd-14-4751-2021
+; See Woodard et al. 2021, 10.5194/gmd-14-4751-2021orh
```
Comment (bpbond) [inline inst/input/hector_ssp126.ini:54]: This seems wrong...the old DOI resolves, but this new one does not.

**Review by bpbond — CHANGES_REQUESTED** [review]
Two minor commenting issues to consider.

Also, do the SSPs vary in their aerosol emissions? I assume yes, and if so might be useful to document if these changes have a bigger effect in e.g. ssp585.

**Comment by ssmithClimate** [comment]
^ Good point on SSPs, worth testing. I'd say just compare new/old, but if the total forcing is not that much difference could just be a comment in text - to keep the extra work down. (The nature of this change is that there's offsetting positive and negative forcing impacts, so its only going to be a pretty specific circumstance where this makes a large difference.).

File: inst/include/forcing_component.hpp (hunk 1 above)
Comment (kdorheim) [reply inst/include/forcing_component.hpp:123]: Up there are a whole bunch of Dorheim et al. in prep citations after publication

Comment (kdorheim) [reply inst/include/forcing_component.hpp:123]: See issue #725

File: inst/input/hector_ssp126.ini (hunk 2 above)
Comment (kdorheim) [reply inst/input/hector_ssp126.ini:54]: That is 100% wrong most likely a fat-fingered typo on my part! Thanks for catching this

**Comment by kdorheim** [comment]
Across the SPPs the total changes on RF total/temp are really small! 

Total RF 
image
MSE for total RF
|scenario               |       MSE|
|:----------------------|---------:|
|hector_ssp119.ini      | 0.0001736|
|hector_ssp126.ini      | 0.0001539|
|hector_ssp245.ini      | 0.0002284|
|hector_ssp370.ini      | 0.0001664|
|hector_ssp434.ini      | 0.0001986|
|hector_ssp460.ini      | 0.0002071|
|hector_ssp534-over.ini | 0.0002214|
|hector_ssp585.ini      | 0.0001971|

Global Temp 
image

MSE Global Temp 
|scenario               |      MSE|
|:----------------------|--------:|
|hector_ssp119.ini      | 7.15e-05|
|hector_ssp126.ini      | 6.13e-05|
|hector_ssp245.ini      | 9.69e-05|
|hector_ssp370.ini      | 5.66e-05|
|hector_ssp434.ini      | 7.46e-05|
|hector_ssp460.ini      | 7.83e-05|
|hector_ssp534-over.ini | 8.39e-05|
|hector_ssp585.ini      | 7.36e-05|


RF BC 
image

MSE RF BC 

|scenario               |       MSE|
|:----------------------|---------:|
|hector_ssp119.ini      | 0.0007222|
|hector_ssp126.ini      | 0.0007518|
|hector_ssp245.ini      | 0.0010348|
|hector_ssp370.ini      | 0.0030172|
|hector_ssp434.ini      | 0.0013467|
|hector_ssp460.ini      | 0.0016318|
|hector_ssp534-over.ini | 0.0010274|
|hector_ssp585.ini      | 0.0012057|

**Comment by bpbond** [comment]
Is MSE mean square error? If so I'm a little confused by that, but anyway, trust your judgment. Thanks!

**Review by bpbond — APPROVED** [review]

**Review by ssmithClimate — APPROVED** [review]

**Pull request merged**